.github/workflows/continuous-integration.yaml

```diff
@@ -35,8 +35,9 @@ jobs:
         restore-keys: ${{ runner.os }}-m2-
     - name: run tests
       run: |
-        source /usr/bin/mvn_test.sh
-        mvn_test
+        mkdir -p /tmp/m2
+        chown -R nobody:nobody . /tmp/m2
```

Comment:
you have no more "regular" user? 
and you shouldn't run any quattor test as root, in any repo.

Reply:
It's the most regular user present in the container but I can create a fresh one if you like.

I will have a look. I think we run as root in most other tests.

Reply:
nobody is not a regular user, it's normally has not even a login shell